src/CCPACSPoint.h

```diff
@@ -24,10 +24,12 @@ namespace tigl
 class CCPACSPoint : public generated::CPACSPoint
 {
 public:
-    TIGL_EXPORT CCPACSPoint(CCPACSControlSurfaceStep* parent, CTiglUIDManager* uidMgr);
-    TIGL_EXPORT CCPACSPoint(CCPACSControlSurfaceHingePoint* parent, CTiglUIDManager* uidMgr);
-    TIGL_EXPORT CCPACSPoint(CCPACSPointList* parent, CTiglUIDManager* uidMgr);
-    TIGL_EXPORT CCPACSPoint(CCPACSSeatModule* parent, CTiglUIDManager* uidMgr);
+
+    TIGL_EXPORT CCPACSPoint(generated::CPACSBoundingBox* parent, CTiglUIDManager* uidMgr);
+    TIGL_EXPORT CCPACSPoint(generated::CPACSControlSurfaceHingePoint* parent, CTiglUIDManager* uidMgr);
+    TIGL_EXPORT CCPACSPoint(generated::CPACSControlSurfaceStep* parent, CTiglUIDManager* uidMgr);
+    TIGL_EXPORT CCPACSPoint(generated::CPACSDeckElementMass* parent, CTiglUIDManager* uidMgr);
+    TIGL_EXPORT CCPACSPoint(generated::CPACSPointList* parent, CTiglUIDManager* uidMgr);
```

Comment:
Your code is actually fine. But just to understand: Why did you change anything here? It should have worked with `CCPACSPointList`, which is just an alias for `genrated::CPACSPointList`, see here:

https://github.com/DLR-SC/tigl/blob/f6f1e4c9042be2232d036f24c82e69b1363b7ab8/src/generated/CPACSPointList.h#L87

```suggestion
    TIGL_EXPORT CCPACSPoint(CCPACSPointList* parent, CTiglUIDManager* uidMgr);
```

The same holds for `CCPACSControlSurfaceStep` and `generated::CPACSControlSurfaceStep`. For consistency, you could change the new constructors to use the `CCPACSXXX` classes as well. These are either customized classes deriving the `generated::CPACSXXX` classes, or just a type def, depending on what was specified in `CustomTypes.txt` before starting the code generator.

Reply:
Thanks for the annotion, you are right. I changed the definitions back to remain the old version